chain-prometheus-exporter/scripts/build.Dockerfile

```diff
@@ -14,3 +14,10 @@ RUN apt-get update && \
     && rm -rf /var/lib/apt/lists/*
 COPY --from=build /build/chain-prometheus-exporter/target/release/chain-prometheus-exporter /usr/local/bin/
 
+WORKDIR /app
+RUN useradd -U -d /app appuser
+RUN chown appuser:appuser /app
+USER appuser
+
+ENTRYPOINT ["chain-prometheus-exporter"]
```

Comment:
I prefer not to add entrypoints and have this configured by the runner instead.

Reply:
I am fine with that, however I do think it is a bit of a shame we diverge from how we define our dockerfiles elsewhere.
`
docker pull 192549843005.dkr.ecr.eu-west-1.amazonaws.com/concordium/collector-backend:1.1.2-0 && docker history --no-trunc 192549843005.dkr.ecr.eu-west-1.amazonaws.com/concordium/collector-backend:1.1.2-0`

provides me with 

```
❯ docker pull 192549843005.dkr.ecr.eu-west-1.amazonaws.com/concordium/collector-backend:1.1.2-0 && docker history --no-trunc 192549843005.dkr.ecr.eu-west-1.amazonaws.com/concordium/collector-backend:1.1.2-0
1.1.2-0: Pulling from concordium/collector-backend
Digest: sha256:3a6da074541232e6408b56871c0121107b4a06041cf376f4cc5e2751866965c0
Status: Image is up to date for 192549843005.dkr.ecr.eu-west-1.amazonaws.com/concordium/collector-backend:1.1.2-0
192549843005.dkr.ecr.eu-west-1.amazonaws.com/concordium/collector-backend:1.1.2-0
IMAGE                                                                     CREATED         CREATED BY                                                                                                                 SIZE      COMMENT
sha256:08064d95fdefed776f62d44ca14abd372cd7fdbf9b4606aae1b2763fe4f3260e   19 months ago   /bin/sh -c #(nop)  LABEL build_type=release                                                                                0B
<missing>                                                                 19 months ago   /bin/sh -c #(nop)  LABEL base_image_tag=latest                                                                             0B
<missing>                                                                 19 months ago   /bin/sh -c #(nop)  ENTRYPOINT ["/node-collector-backend"]                                                                  0B
...
file:5d68d27cc15a80653c93d3a0b262a28112d47a46326ff5fc2dfbf7fa3b9a0ce8 in /                           72.8MB
```

where `node-collector-backend` calls the binary that has been built, which is the same as I am doing here.

I can also see we do exactly the same on the `ccd-scan` for example.

Also it is rather common that one uses ENTRYPOINT or CMD in order to specify the default behaviour of the image over the caller having to look up where the binary lies in the image order to call it.

For now I will specify an entrypoint in the deployment to get https://concordium.atlassian.net/jira/software/c/projects/CD/boards/17?modal=detail&selectedIssue=CD-587 done, however I still think this discussion is relevant if we do not agree upon the same contract when specifying different dockerfiles.